app/swagger/schemas/vet360/contact_information.rb

```diff
@@ -287,6 +287,32 @@ class ContactInformation
                            format: 'date-time',
                            example: '2018-04-21T20:09:50Z'
                 end
+
+                property :text_permission, type: :object do
+                  property :id, type: :integer, example: 123
+                  property :permission_type, type: :string, example: ::Vet360::Models::Permission::TEXT
+                  property :permission_value, type: :boolean, example: true
+                  property :created_at,
+                           type: :string,
+                           format: 'date-time',
+                           example: '2019-00-23T20:09:50Z'
+                  property :effective_end_date,
```

Comment:
similar to the comment above, the controller was already explicitly removing this attribute, so it should probably be removed from the documentation as well, since it is ignored.